src/data/pokemon/item_effects.h

```diff
@@ -36,20 +36,12 @@ const u8 gItemEffect_MaxPotion[7] = {
 
 const u8 gItemEffect_HyperPotion[7] = {
     [4] = ITEM4_HEAL_HP,
```

Comment:
Out of scope for this PR obviously, but I really feel like the array indices should not be magic numbers supported by a comment and rather be easier through use through defines or macros or something

Reply:
If someone is able to do so, they should PR it to pret :)